### PR TITLE
#244 Test editRisk endpoint successes

### LINKED ISSUE #244: Backend - Test editRisk Endpoint Successes
### Description

Write unit tests for the successful cases of the editRisk endpoint. To work on this ticket, you will need to read through the endpoint to learn how it works.

### Acceptance Criteria

The following cases should be covered:
* the original risk was resolved and the payload is trying to unresolve it
* the original risk wasn't resolved and the payload is trying to resolve it
* the original risk and payload have the same resolved value

### Proposed Solution

Check out this epic https://github.com/Northeastern-Electric-Racing/FinishLine/issues/238 for resources and tips on how to write these tests.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by jaredlyon**
## Changes

Created tests checking the endpoints of successful risk edits. Complementary test data has also been added.

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #244

### CODE REVIEW

**Review by anthonybernardi — CHANGES_REQUESTED** [review]
you need to mock any calls to prisma that are made

File: src/backend/tests/risks.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  detail: 'This one might be a bit too expensive',
+  resolved: true
+};
+
+const editRiskFalse = {
+  userId: 1,
+  id: '4800c304-2f8d-4376-a4c6-d7a157134c83',
+  detail: 'This one might be a bit too expensive',
+  resolved: false
+};
```
Comment (rishavsarma5) [inline src/backend/tests/risks.test.ts:21]: move these to a test-data file and import them into this test file

Comment (jaredlyon) [reply src/backend/tests/risks.test.ts:21]: 👍🏽

**Comment by jaredlyon** [comment]
Screen Shot 2022-10-26 at 8 01 27 PM

@anthonybernardi --> Reid and I worked on this ticket today and ran into an issue; specifically, commenting out the `jest.spyOn` calls allows the test to pass like originally shown in [2b0b153](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/330/commits/2b0b1533252d615b696a6ac4f01a3dfa9bbb5fc3), but we were unable to figure out a way to pass in the correct objects for `jest.spyOn`. Any ideas on what's going on here?

**Comment by jaredlyon** [comment]
Finalized code is up - should be ready for review (again).

**Review by anthonybernardi — APPROVED** [review]
awesome work!!!

**Pull request merged**